my_list.rb

```diff
@@ -0,0 +1,13 @@
+require('./MyEnumerable')
```

Comment:
- [ Required ] I think it would be better if you included this file in your project using the `require_relative` keyword, ruby has a convention that ruby gems ought to be imported into another file using the `require` statement, however, you need to use the `require_relative` when importing ruby files that you have authored. 
- [ Required ] I also encountered the error below when running your project, I believe this is because you are importing the `MyEnumerable` file which is not present in your project directory it should be `my_enumerable`. Please rectify this.

## Example code
```ruby
 require_relative 'my_enumerable'
```

## Error
![Screenshot from 2023-05-19 22-39-42](https://github.com/BahirHakimy/enumerable-ruby/assets/16392046/4675dc6f-97fc-4974-bcbb-853d60abdaaa)

Reply:
Thanks for your feedback @Felix45 
I will fix the issues